locales/ja.json

```diff
@@ -0,0 +1,289 @@
+{
+  "components": {
+    "airport_select": {
+      "placeholder": "空港の選考"
+    },
+    "color_select": {
+      "placeholder": "色彩の選考"
+    },
+    "line_index_select": {
+      "custom_indices": "路線記号の選考",
+      "line_name": "{index}線",
+      "official_indices": "公式の路線記号",
+      "placeholder": "特注の路線記号"
+    },
+    "map_size_select": {
+      "placeholder": "サイズの選考"
+    },
+    "mode_select": {
+      "placeholder": "交通機関の選考"
+    },
+    "service_select": {
+      "placeholder": "交通サービスの選考"
+    },
+    "shape_select": {
+      "placeholder": "形の選考"
+    }
+  },
+  "data": {
+    "airports": {
+      "both": "CDG + ORY",
+      "cdg": "パリ＝シャルル・ド・ゴール空港",
+      "generic": "空港",
+      "ory": "パリ＝オルリー空港"
+    },
+    "colors": {
+      "custom": "特注の色"
+    },
+    "map_size": {
+      "big": "XL (30em)",
+      "huge": "XXL (50em)",
+      "large": "L (20em)",
+      "normal": "M (15em)"
+    },
+    "mode": {
+      "bike": "自転車",
+      "boat": "水上バス",
+      "brt": "高速バス",
+      "bus": "バス",
+      "cable": "ロープウェイ",
+      "metro": "地下鉄",
+      "rer": "RER",
+      "tram": "路面電車",
+      "transilien": "Transilien"
+    },
+    "services": {
+      "airport": "空港",
+      "bullet_train": "新幹線",
+      "cdgval": "CDGVAL",
+      "funicular": "ケーブルカー",
+      "long_distance_bus": "高速バス",
+      "main_station": "主要駅",
+      "orlyval": "OrlyVal",
+      "suburban_train": "通勤列車",
+      "ter": "TER",
+      "tgv": "TGV"
+    },
+    "shapes": {
+      "circle": "丸",
+      "lines": "線",
+      "rectangle": "長方形",
+      "rounded_square": "角丸四角形"
+    }
+  },
+  "locales": {
+    "en": "英語",
+    "fr": "フランス語",
+    "ja": "日本語"
+  },
+  "misc": {
+    "beta": "ベータ版",
```

Comment:
Maybe leave this in Latin alphabet "Beta"?

Reply:
According to jisho.org and Japanese Wikipedia, the katakana version seems the one in use.

Reply:
Alright, fair enough